index.bs

```diff
@@ -112,8 +114,24 @@ The Magnetometer Interface {#magnetometer-interface}
   };
 </pre>
 
-To <dfn>Construct a Magnetometer Object</dfn> the user agent must invoke the
-<a>construct a Sensor object</a> abstract operation.
+
+The RawMagnetometer Interface {#raw-magnetometer-interface}
```

Comment:
I am not a big fan on this Raw class... then I guess Accelerometer and Gyroscope are considered raw as well? Or do we need to expose raw sensors for those as well? It seems messy.

I rather have a raw class called Magnetomer and a higher level one called something like MagneticFieldSensor or similar depending how what it actually is used for. Sensors without Sensor in their names are generally quite low level, so far at least